src/test/java/org/apache/commons/lang3/builder/CompareToBuilderTest.java

```diff
@@ -884,9 +884,9 @@ public void testMultiBooleanArray() {
 
     @Test
     public void testRaggedArray() {
-        final long array1[][] = new long[2][];
```

Comment:
This array declaration is called "C-type array declaration" and it is not recommended.